doc/changes/latest.inc

```diff
@@ -34,5 +34,7 @@ Bug
 
 - Fix IO of TFRs when event_id contain a / in one of the keys by `Alex Gramfort`_
 
+- Fix threshold setting in :func:`mne.preprocessing.ica.corrmap` by `Fahimeh Mamashli`_ and `Eric Larson`_
+
```

Comment:
Can you remove these lines? Should be part of the other PR

Reply:
@fmamashli did you branch from `master` ? :)